### PR TITLE
Enhancements to Context Mapping repo

### LINKED ISSUE #5: 2 levels Context Maps
Hello @mploed 

I was wondering if you had already seen or done such a thing: having a 2 levels context maps where 1st level would focus only on team relationships and 2nd level, only on context map patterns.

Here's an example:
(and yes, obviously the Cus-->Sup relationship should be the other way round ;) )

**Team relationships**
Team

**Context map patterns**
ContextMap

**Both**
Both

What do you think? Does it make sense or useful to separate both perspective? 

Thanks!

Maxime

### PR DESCRIPTION
**Pull request by mploed**
I want to add the following things to the context-mapping repo:

- Brief description of the patterns as an overview, mostly taken from the DDD Reference (which is also licensed as creative commons)
- An overview for different perspectives of context maps. Mostly based on the issue raised by @Max-Git 
- Some best practices and hints

### CODE REVIEW

**Comment by NTCoding** [comment]
Looks good so far. The text descriptions are clear and the images look good.

One thing you may wish to do is to update the link to the Miro board backup to an actual Miro board. It's now possible to copy and paste from Miro boards even for guests with read only access when the owners of the Miro board enables sharing: https://help.miro.com/hc/en-us/articles/360018350399.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,123 @@ Context Maps describe the contact between bounded contexts and teams with a coll
...
+
+"Designate with an explicit boundary some subset of the domain model that the teams agree to share. Keep this kernel small.Within this boundary, include, along with this subset of the model, the subset of code or of the database design associated with that part of the model. This explicitly shared stuff has special status, and shouldn’t be changed without consultation with the other team." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
+#### Partnership
+
+<img src="resources/partnership.jpg" alt="Partnership" width=300/>
+
+"Where development failure in either of two contexts would result in delivery failure for both, forge a partnership between the teams in charge of the two contexts. Institute a process for coordinated planning of development and joint management of integration. 
+The teams must cooperate on the evolution of their interfaces to accommodate the development needs of both systems. Interdependent features should be scheduled so that they are completed for the same release." ([Source: DDD Reference by Eric Evans](https://www.domainlanguage.com/ddd/reference/))
+
```
Comment (hemalvarambhia) [inline README.md:65]: Interestingly I couldn't find 'Partnership' in the blue book. Could you? If it really isn't there, this relationship was named after 2004.

Comment (mploed) [reply README.md:65]: It was added by Vaughn Vernon in the red book and later added to the DDD Reference by Eric Evans.

Comment (hemalvarambhia) [reply README.md:65]: That is interesting. Perhaps worth adding (red book) as an additional reference? Non blocking suggestion

**Comment by hemalvarambhia** [comment]
@mploed just forgot to mention: this looks really great, a resource I would certainly love to share with my colleagues.

**Comment by ChristinaHirth** [comment]
I just realised that neither this nor #6 was ever merged. Maybe it is time now @mploed @hemalvarambhia @NTCoding 😉 ?

**Comment by hemalvarambhia** [comment]
> I just realised that neither this nor #6 was ever merged. Maybe it is time now @mploed @hemalvarambhia @NTCoding 😉 ?

@ChristinaHirth go for it!

**Comment by mploed** [comment]
Merged it ;-)

**Review by hemalvarambhia — APPROVED** [review]

**Pull request merged**